Automattic/vip-cli
# SQL Imports: Simplify import update progress logs

## Description

Simplifying the user output logs for the SQL import progress.
We also want the output to visually match our other processes (e.g.- data sync).

## Steps to Test

1. Check out PR.
1. Run `npm run build`
1. Run `node ./dist/bin/vip-import-sql @103.staging /path/to/file.sql --search-replace="hello.com,goodbye.com"`
1. Verify update progress logs look good
1. Try without the S-R flag, and verify that you don't see any of the S-R-related output
